deploy/monitoring/csi-pod-monitor.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Could we place here simple instructions how to enable Openshift monitoring or a link to them too?

Reply:
Sure, done.

Reply:
thanks!